src/common/XAuth.cpp

```diff
@@ -99,29 +103,61 @@ bool XAuth::addCookie(const QString &display)
 }
 
 bool XAuth::addCookieToFile(const QString &display, const QString &fileName,
-                            const QString &cookie)
+                            const QByteArray &m_cookie)
```

Comment:
This shadows the member variable.

Either use `cookie` here or remove the parameter.